directory-manifest/directory-listing.xsl

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  xmlns="http://www.w3.org/1999/xhtml"
+  xmlns:p="http://www.w3.org/ns/xproc"
+  xmlns:c="http://www.w3.org/ns/xproc-step"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:err="http://www.w3.org/2005/xqt-errors"
+  version="3.0"
+  exclude-result-prefixes="#all"
+  expand-text="true">
+
+  <!-- Purpose: Produce an HTML report from polling a set of XML files provided as a directory list -->
+  <!-- Input: A c:directory document such as is delivered by XProc `p:directory-list` step -->
+  <!-- Limitations: Built out only for common cases seen so far, YMMV -->
+  <!-- Note: Logic is extensible to handle analysis/synopsis of any XML document type -->
+
+  <xsl:strip-space elements="*"/>
+  
+  <xsl:variable name="path" select="/c:directory/@xml:base"/>
+  
+  <xsl:template match="/">
+    <html>
+      <head>
+        <title>Directory Manifest: { $path }</title>
+      </head>
+      <body data-path="{$path}">
+        <h1>Directory Manifest: <code>{ tokenize($path,'/')[last() - 1] }</code></h1>
+        <p class="timestamp">{ current-dateTime() => format-dateTime('[MNn] [D0] [Y] [h0]:[m01] [P]') } - { current-dateTime() } -</p>
```

Comment:
That's a good idea, but how should the switch look to turn it on?

For now, it can be easily edited out of the file by hand.

For later, if you think this should be switched on only at invocation, make an Issue and we can sketch what that would look like. The switch is about the easiest switch to implement but we will have to think about the scripts also.

Reply:
OK I was just arguing from the perspective of making it easily consumable as-is.